ci/jenkins/test-vmc.sh

```diff
@@ -454,8 +452,9 @@ function run_integration {
     echo "===== Run Integration tests ====="
     # umask ensures that files are cloned with the correct permissions so that Docker caching can be leveraged
     ${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "umask 0022 && git clone ${ghprbAuthorRepoGitUrl} antrea && cd antrea && git checkout ${GIT_BRANCH} && DOCKER_REGISTRY=${DOCKER_REGISTRY} ./build/images/ovs/build.sh --pull && NO_PULL=${NO_PULL} make docker-test-integration"
+    SHA=$(${SSH_WITH_UTILS_KEY} -n jenkins@${VM_IP} "cd antrea && git rev-parse HEAD")
```

Comment:
should the `run_codecov` shell function take care of computing the SHA to simplify call sites?

Reply:
it will be more complicate to determine the SHA in `run_codecov` because different cases call this method in remote or local and in different git directory. so I just pass a parameter.

Reply:
A follow up question: can we just use the `GIT_COMMIT` env variable provided by the Jenkins Git plugin? I see that we already use `GIT_BRANCH` in this script.

Reply:
sure, let's update and verify it. thanks.

Reply:
done.

Reply:
I don't see the change?

Reply:
sorry, I just do another force push, maybe there is an issue last time and I didn't notice it.